src/main/java/teammates/common/datatransfer/FeedbackSessionQuestionsBundle.java

```diff
@@ -113,8 +113,9 @@ public void hideUnmoderatableQuestions() {
 
             if (!isResponseVisibleToInstructor || !isGiverVisibleToInstructor || !isRecipientVisibleToInstructor) {
                 questionsToHide.add(question);
-                questionResponseBundle.put(question, new ArrayList<FeedbackResponseAttributes>());
+
             }
+            questionResponseBundle.putIfAbsent(question, new ArrayList<>());
```

Comment:
Its nice that you are using diamond expression to shorten the code, but I think this change is unnecessary as we do not need to check if this question exist in the bundle or not, after all these `questions` will be removed from the `questionResponseBundle` in the next line.